shared-brotli-patch-decoder/Cargo.toml

```diff
@@ -11,8 +11,16 @@ repository.workspace = true
 # RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --all-features
 all-features = true
 
+[features]
+default = ["c-brotli"]
```

Comment:
Optional, I wonder if rust-brotli isn't a more natural default for a rust program?

Reply:
A couple of things I'd like to resolve first before making rust the default:
1. Need to test the relative performance of the two versions. If rust is comparable to c then it should be fine as a default.
2. Need to fix the inability of the rust version to detect an input that has unused trailing data, since that's required by the spec.